metrics/web/test/features/dashboard/presentation/widgets/placeholder_bar_test.dart

```diff
@@ -0,0 +1,52 @@
+// https://github.com/software-platform/monorepo/issues/140
+// ignore_for_file: prefer_const_constructors
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/colored_bar.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/placeholder_bar.dart';
+
+void main() {
+  const _width = 2.0;
+
+  group("PlaceholderBar", () {
+    testWidgets(
+      "displays the ColoredBar",
```

Comment:
What if we add tests that will be understandable and closer to the specification/current understanding? Here are some examples:
"displays as short box" -> check container height
"displays as a bar with grey border" -> check ColoredBar border?